quarkc/test/ffi/expected/py/package/docs/conf.py

```diff
@@ -1,16 +1,16 @@
 # -*- coding: utf-8 -*-
 #
-# package documentation build configuration file, created by Quark
+# package_ documentation build configuration file, created by Quark
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.napoleon'
 ]
 templates_path = ['_templates']
 source_suffix = '.rst'
 master_doc = 'index'
-project = u'package'
-copyright = u'2015, package authors'
-author = u'package authors'
+project = u'package_'
```

Comment:
As a user I would rather the compiler told me "don't use `package` as identifier" over the compiler randomly changing my public API in unexpected ways.